<ui version="4.0" >
 <author>Copyright (c) 2006 Fabrizio Pollastri</author>
 <class>example</class>
 <widget class="QMainWindow" name="example" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AVC Qt4 showcase example</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>658</width>
     <height>747</height>
    </rect>
   </property>
   <widget class="QPushButton" name="boolean1__pushbutton" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>70</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text" >
     <string>push button</string>
    </property>
   </widget>
   <widget class="Line" name="line" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boolean2__pushbutton2" >
    <property name="geometry" >
     <rect>
      <x>161</x>
      <y>120</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text" >
     <string>push button</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boolean2" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="text" >
     <string>check box</string>
    </property>
   </widget>
   <widget class="Line" name="line_2" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>160</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio__button0" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>210</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>choice 0</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio__button2" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>270</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>choice 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>180</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>radio button</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio__button1" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>240</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>choice 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>180</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>combo box</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>320</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>spin box</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="float__entry" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>390</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>320</y>
      <width>32</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>entry</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="integer__spinbox" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>350</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>999999</number>
    </property>
   </widget>
   <widget class="Line" name="line_4" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>430</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>450</y>
      <width>32</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>entry</string>
    </property>
   </widget>
   <widget class="Line" name="line_5" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>610</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>430</x>
      <y>320</y>
      <width>34</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>630</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>text view/edit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="integer__entry" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>350</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="string__entry" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>480</y>
      <width>280</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>300</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textview__textedit" >
    <property name="geometry" >
     <rect>
      <x>203</x>
      <y>660</y>
      <width>281</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="radio__combobox" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>230</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumContentsLength" >
     <number>0</number>
    </property>
    <item>
     <property name="text" >
      <string>choice 0</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>choice 1</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>choice 2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="textLabel1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Control Type</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Widgets</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>20</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Control Value</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>100</y>
      <width>108</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>boolean</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel5" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>230</y>
      <width>108</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>index (integer)</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>350</y>
      <width>108</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>integer</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>390</y>
      <width>108</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>float</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel8" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>480</y>
      <width>108</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>string</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>690</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>string</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textview__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>666</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>text view</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="string__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>480</y>
      <width>102</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>string</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="integer__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>350</y>
      <width>102</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>integer</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="radio__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>220</y>
      <width>102</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>index</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="boolean2__label" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>120</y>
      <width>102</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>boolean2</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="boolean1__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>70</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>boolean1</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="integer__slider" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="singleStep" >
     <number>1</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="float__spinbox" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>390</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="specialValueText" >
     <string/>
    </property>
    <property name="decimals" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <double>1000000.000000000000000</double>
    </property>
    <property name="singleStep" >
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="float__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>390</y>
      <width>102</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>%.1f</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>520</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>540</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>progress bar</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressbar__progressbar" >
    <property name="geometry" >
     <rect>
      <x>207</x>
      <y>570</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel7_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>570</y>
      <width>108</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>float</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="progressbar__var" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>570</y>
      <width>102</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>%.2f</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
